### PR TITLE
10-final_SerhiiNSN

### PR DESCRIPTION
**Pull request by SerhiiNSN**
Link to deployed project - https://serhiinsn.github.io/fe-course-basic-2023/

### CODE REVIEW

File: docs/SerhiiNSN/scripts/burger.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const burger = document.querySelector('.header__burger');
+const menu = document.querySelector('.nav');
+const body = document.querySelector('body');
+burger.addEventListener('click', () => {
+    burger.classList.toggle('active');
```
Comment (vladyslav-yermolin-moc) [inline docs/SerhiiNSN/scripts/burger.js:5]: Proly you can find more suitable names for these classes, like `opened`, `no-scroll` e.g.

File: docs/SerhiiNSN/scripts/games.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+function showLoadingOverlay() {
+    loadingOverlay.style.display = 'block';
+}
+
+function hideLoadingOverlay() {
+    loadingOverlay.style.display = 'none';
+}
+
+async function createCardElement(game) {
```
Comment (vladyslav-yermolin-moc) [inline docs/SerhiiNSN/scripts/games.js:16]: Why do you make this function `async`? There is no async operation in it.

File: docs/SerhiiNSN/scripts/games.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    cardCopy.querySelector('[data-card-release]').innerHTML = `<span class='bold'>Release_date:</span> ${game.release_date}`;
+
+    return cardCopy;
+}
+
+async function renderCards(container, games) {
+    const fragment = document.createDocumentFragment();
+    const containerElement = document.querySelector(`[data-type="${container}"]`);
+
+    await Promise.all(games.map(async (game) => {
```
Comment (vladyslav-yermolin-moc) [inline docs/SerhiiNSN/scripts/games.js:32]: ... then you can skip this `await Promise.all` and make the current function synchronous too.

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]
Good work 👍

File: docs/SerhiiNSN/games.html [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+                                    <div class="card__tittle" data-card-title>
+                                        Overwatch 2
+                                    </div>
+                                    <div class="card__text">
+                                        Big changes come to the Overwatch
+                                        formula in this sequel...
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="card__about">
```
[the remark this answers is not in the document]
Comment (vladyslav-yermolin-moc) [reply docs/SerhiiNSN/games.html:140]: This `line-height: 150%` makes the text row taller 
image
image

[the remark this answers is not in the document]
Comment (vladyslav-yermolin-moc) [reply docs/SerhiiNSN/games.html:140]: And add please decent class name https://github.com/MastersAcademy/fe-course-basic-2023/issues/426#issue-2042776096
> Задавайте стилі лише на класи (не на елементи)

**Review by vladyslav-yermolin-moc — DISMISSED** [review]
Good, I can approve this pr, please fix those minor comments in the nearest future

**Review by vladyslav-yermolin-moc — APPROVED** [review]

**Review by MZabolotnev — APPROVED** [review]

**Pull request merged**